requirements/edx/base.txt

```diff
@@ -803,7 +803,7 @@ openedx-filters==1.6.0
     # via
     #   -r requirements/edx/kernel.in
     #   lti-consumer-xblock
-openedx-learning==0.3.4
+openedx-learning @ git+https://github.com/open-craft/openedx-learning.git@rpenido/fal-3536-overwrite-tags-on-import
```

Comment:
openedx-learning==0.3.5 is ready now: https://pypi.org/project/openedx-learning/0.3.5/

Reply:
Done 0d9b82e86d56c219d599be2aa6e076d0a108f2a6!